### ISSUE TITLE
expletive with raising as csubj vs ccomp

### ISSUE THREAD

**Issue by amir-zeldes** [UniversalDependencies/UD_English-EWT#120]
There are two analyses in EWT for raising verbs with expletive "it seems":

  *  it/expl seems that the rest of Baghdad is 'normal'/**ccomp** (weblog-blogspot.com_healingiraq_20040409053012_ENG_20040409_053012-0011)
  * it/expl seems likely that Iraqi Shiites will get rich/**csubj** enough

I think the second one with csubj is better (this is what we have in GUM), but either way I think we should choose one analysis.

**Comment by nschneid** [comment]
I think `csubj` is correct per example 5 on https://universaldependencies.org/u/dep/expl.html

**Comment by nschneid** [comment]
I think it's fair to say that any predicate with an `expl` dependent and a complement must also have some sort of subject.

Here are 27 violations in EWT, some of which aren't actually `expl`: http://match.grew.fr/?corpus=UD_English-EWT@2.7&custom=5fedf4f6bdfb3

The most interesting is "When it came time to pay the bill". "Came time" is idiomatic, probably related to "became". What would the structure be for "It became rainy"—`expl` + `xcomp`?

"it seemed to take the hotel staff quite a while to quiet them down": should be `csubj(seemed, take)`?

"There needs to be a recount"—"There is a recount" would be `expl(is, There)`, `nsubj(is, recount)` (example 7 [here](https://universaldependencies.org/u/dep/nsubj.html)). If the main verb is "needs", I guess the infinitival clause should be `csubj`?: `expl(needs, There)`, `csubj(needs, be)`, `nsubj(be, recount)`?

Also, here's a There+BE+NP existential sentence where the BE verb should be the head: http://match.grew.fr/?corpus=UD_English-EWT@2.7&custom=5fedf761a5c27

**Comment by amir-zeldes** [comment]
Agreed, I think all of these should have non-expl subjects, so these should be csubj based on the following paraphrases:

  * Time to pay the bill came (so "time" is `nsubj`)
  * To quiet them down seemed to take the hotel staff quite a while

For the recount, note that EWT makes lexical subjects of raising verbs be nsubj to the raising verb, not the embedded predicate (unlike SD used to do earlier), so I think that suggests no csubj here, but rather nsubj:

  * There needs to be a recount (="a recount needs to be") -> expl(needs, there), nsubj(needs, recount), xcomp(needs, be)

This is by analogy to lexical subject cases like "Lori seems to think" here:

```
# sent_id = email-enronsent04_01-0006
...
14	Lori	Lori	PROPN	NNP	Number=Sing	15	nsubj	15:nsubj|17:nsubj:xsubj	_
15	seems	seem	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
16	to	to	PART	TO	_	17	mark	17:mark	_
17	think	think	VERB	VB	VerbForm=Inf	15	xcomp	15:xcomp	_
```

(so Lori is not taken to be the syntactic subject of "think", even though semantically that is implied by the raising analysis, and there is no csubj)

**Comment by nschneid** [comment]
Reflecting on the `xcomp` analysis, I think the idea is that a control or raising verb can "steal" the expletive subject from the embedded predicate. So the rule about having no complement without a non-expletive subject then applies to the embedded predicate. I think all of these are incorrect: http://match.grew.fr/?corpus=UD_English-EWT@2.7&custom=5fedfb8215fde

**Comment by nschneid** [comment]
@amir-zeldes For "There needs to be a recount" I'm thinking: expl(needs, there), xcomp(needs, be), nsubj(be, recount). The raising verb can steal an embedded *surface* subject, i.e. "there", but not the *deep* subject "recount", I don't think.

**Comment by nschneid** [comment]
I.e. "deep" structure of

needs [There_expl/arg0 BE a recount_nsubj/arg1]

is realized as

There_expl/arg0 needs [TO-BE a recount_nsubj/arg1]

(where subjects/complements are numbered in order, arg0 being the thing that appears before the verb)

**Comment by amir-zeldes** [comment]
> Reflecting on the `xcomp` analysis, I think the idea is that a control or raising verb can "steal" the expletive subject from the embedded predicate. So the rule about having no complement without a non-expletive subject then applies to the embedded predicate. I think all of these are incorrect: http://match.grew.fr/?corpus=UD_English-EWT@2.7&custom=5fedfb8215fde

I agree, those look wrong to me too:
  * it is crucial to reach the top - should be expl + csubj, not xcomp
  * I leave it to you to figure out - should be expl + ccomp, not xcomp

**Comment by amir-zeldes** [comment]
> @amir-zeldes For "There needs to be a recount" I'm thinking: expl(needs, there), xcomp(needs, be), nsubj(be, recount). The raising verb can steal an embedded _surface_ subject, i.e. "there", but not the _deep_ subject "recount", I don't think.

This is basically the old SD analysis, which I take it was overturned for some reason. I don't feel very strongly about it, but it would kind of annoy me to revert all of these, because we actually went to some trouble moving from SD to UD, to *undo* all of these analyses into the "Lori seems to think" form above (which in SD used to have "think" as the head of "Lori" IIRC). I think you could either prioritize semantics, and say "I want the tree to reflect the semantic argument structure", so what "needs" is "for a recount to be", or say that UD represents surface morphosyntax, and on the surface, agreement suggests that "recount" is the subject, incl. both word order and the 3rd person 's', compare:

  * There needs to be a recount
  * There need to be recounts (need inflects as if "recounts" is its subject)

Since UD doesn't have a 'deep' layer or anything like transformations, you can only do justice to one of these ideas in the basic tree (semantic argument structure or surface morphosyntax). Edeps could let you express both of course.

Maybe @mcdm or @sebschu can chime in for some context on how this changed from SD to UD?

**Comment by nschneid** [comment]
This kind of thing is why I'm not a syntactician. :)

With agreement as a criterion I guess the structure would differ between

- There need to be recounts: expl(need, there), nsubj(**need**, recounts), xcomp(need, be)
- John needs there to be recounts: expl(needs, there), nsubj(needs, John), nsubj(**be**, recounts), xcomp(need, be)

which is awfully subtle.

My guess was that with an `xcomp`, only one shared argument would be attached to the matrix verb, thus

- There need to be recounts: expl(need, there), nsubj(**be**, recounts), xcomp(need, be)
- John needs there to be recounts: expl(needs, there), nsubj(needs, John), nsubj(**be**, recounts), xcomp(need, be)

Agreed that a clarification would be helpful!

**Comment by nschneid** [comment]
Encountering this again in #302—"there seems to be...", "there is going to be...", etc.

**Comment by nschneid** [comment]
OK here's what I think is going on with the above examples, showing a gap in the embedded clause:

## Non-expletive

- These/nsubj are/cop recounts.
- These_i <-nsubj- seem [__i to be <-cop- recounts].

## There-expletive/existential

- There <-expl- are -nsubj-> recounts.
- There_i <-expl- need [__i to be -nsubj-> recounts]. (in matrix clause it is an expletive subject, triggering agreement)
   * expl(need, there), xcomp(need, be), nsubj(be, recounts)
- John <-nsubj- needs -expl-> there_i [__i to be -nsubj->recounts]. (in matrix clause it is an expletive object, as the subject is John)
   * nsubj(needs, John), expl(need, there), xcomp(needs, be), nsubj(be, recounts)

## It-expletive

### Infinitival extraposed clause

- It/expl seems [to be a good idea]/xcomp [for you to resign]/csubj.
- [For you to resign]/csubj seems [(to be) a good idea]/xcomp.

### Finite extraposed clause

- It <-expl- seems [(that) these are recounts]
- *That these are recounts seems.
  * Normally with extrapositions, the full phrase can be substituted for expletive "it". @amir-zeldes suggests (pc) that this is an obligatory extraposition, so the subordinate clause is csubj.
  * Perhaps another way to characterize the extraposition is as an ellipsis for:
- It seems [to be the case] [(that) these are recounts].
- [That these are recounts]/csubj seems [to be the case]/xcomp.
   * N.B.: *These are recounts seems to be the case. ("That" is obligatory. I guess a sentence-initial finite csubj cannot start with its subject to avoid garden-pathing.)

## Comparative-like cases

- It seems like these are recounts.
- It seems as though these are recounts.
- *Like these are recounts seems to be the case.
- *As though these are recounts seems to be the case.
- *There seem like to be recounts.
- *There seem as though to be recounts.

Even if these sentences are not read with comparative semantics, arguably the drawing-a-conclusion construction recruits the comparative clause strategy. I wonder if advcl is appropriate for the like/as clause (but then does "seem" have any core arguments?).

**Comment by amir-zeldes** [comment]
I like csubj even for the like cases - as you say it saturates seem's argument, and otherwise we split it into two kinds of "seem" just based on the presence of "like". I think a marked clause can assume the subject position similarly to a PP, e.g. "at home is perfect". If we like "home" as a subject in that case (assuming a PP to NP conversion via unary derivation), then "like" as a subject clause should also be possible.

**Comment by nschneid** [comment]
Yes PPs can be subjects. I dunno if that implies contentful-subordinator-marked clauses can be as well though.

- ?Because he arrived late is a bad reason to punish him.
- ?If you are sick is the only good reason for missing class.

I can imagine people saying these in conversation, but do we want to treat them as grammatical, or as grammatical gear-shifting mid-sentence?

**Comment by rueter** [comment]
> Yes PPs can be subjects. I dunno if that implies contentful-subordinator-marked clauses can be as well though.
> 
> - ?Because he arrived late is a bad reason to punish him.
> - ?If you are sick is the only good reason for missing class.
> 
> I can imagine people saying these in conversation, but do we want to treat them as grammatical, or as structural gear-shifting mid-sentence?

The first of the two examples sounds like a mention, i.e., "'Because he arrived late' is a bad reason to punish him."
The second example, however, is hard to really imagine.

**Comment by amir-zeldes** [comment]
> I can imagine people saying these in conversation, but do we want to treat them as grammatical, or as structural gear-shifting mid-sentence?

I think it's grammatical to do that, at least to the extent that I wouldn't feel that some sort of `reparandum` analysis is warranted there. If we think the "because" example should receive a proper analysis, then that's already reason enough to allow it.
